visualdl/logic/pybind.cc

```diff
@@ -219,6 +233,61 @@ PYBIND11_MODULE(core, m) {
       .def("total_records", &cp::TextReader::total_records)
       .def("size", &cp::TextReader::size);
 
+  py::class_<cp::Audio>(m, "AudioWriter", R"pbdoc(
```

Comment:
Will it be weird to have documentations published on the website but not the code is not in the release pip? I am not sure what's the best approach here.